runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -36,6 +36,11 @@
 import java.io.File;
 import javax.inject.Singleton;
 import joptsimple.ArgumentAcceptingOptionSpec;
+
+import javax.swing.JLabel;
```

Comment:
Fix the imports based on the codestyle wiki page

Reply:
Oh yeah think I might have accidentally added the new line when rehashing